designs/2021-suppression-support/README.md

```diff
@@ -7,7 +7,7 @@
 
 ## Summary
 
-Violations can be suppressed by inline directive comments and external configuration files. We propose to add an `--track-suppressions` CLI option for adding suppression information, including its kind and justification, in the output of ESLint.
+Violations can be suppressed by inline directive comments and external configuration files. We propose to add a `--track-suppressions` CLI option for adding suppression information, including its kind and justification, in the output of ESLint. We also propose to add a new severity 3 - `suppress` to suppress (not skip) rules in configuration files.
```

Comment:
It’s a bit weird that 0, 1, 2 are in order of increasing severity, but 3 is more like 0.5